src/Parser/HolyPriest/CPM_ABILITIES.js

```diff
@@ -26,14 +26,6 @@ const CPM_ABILITIES = [
     getMaxCasts: (cooldown, fightDuration, getAbility, parser) => {
       return calculateMaxCasts(cooldown, fightDuration, 2);
     },
-    recommendedCastEfficiency: 0.50, // This spell is a very odd use, most people use it during movement or for tanks during heavy damage, so this does not really need to be super high but it needs to be high enough that you dont sit at 2 charges
-  },
-  {
```

Comment:
This also goes for cooldowns like Pain Supp; some fights it isn't bad to only have cast it once, but being able to see that you could have cast it more often in the cast efficiency tab should get people to reflect on that instead when it's not being shown to them and they're not thinking about it.